src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -295,21 +295,26 @@ private static PackageReference ToPackageReference(LibraryDependency library, Nu
                 .GetPackageReferencesAsync(targetFramework, CancellationToken.None))
                 .ToList();
 
-            var packageTargetFallback = _vsProjectAdapter.PackageTargetFallback?.Split(new[] { ';' })
+            var splitChars = new[] { ';' };
+
+            var packageTargetFallback = _vsProjectAdapter.PackageTargetFallback?.Split(splitChars)
                 .Select(NuGetFramework.Parse)
-                .ToList();
+                .ToList()
```

Comment:
NuGetFramework.Parse returns the Unsupported framework since it isn't unusual for future frameworks to come in. For fallback frameworks there is a check to verify that unsupported isn't passed and it will throw.